src/renderer/services/dash/balance.ts

```
@@ -1,7 +1,36 @@
+      return keystoreClient
+    }
+
+    // If keystore is locked but we're in standalone ledger mode, use read-only client
+    if (appWalletState && isStandaloneLedgerMode(appWalletState) && O.isSome(readOnlyClient)) {
+      return readOnlyClient
+    }
+
+    // Otherwise, no client available
+    return O.none
+  }),
+  RxOp.distinctUntilChanged((a, b) => O.getEq({ equals: (x: any, y: any) => x === y }).equals(a, b)),
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)